### PR TITLE
PMU microcode firmware interface documentation

### PR DESCRIPTION
**Pull request by Echelon9**
Notes from the blob PMU (PGRAPH) Falcon microcode firmware that ships with NVIDIA 390.67.

Comprises:
* MMIO-side registers to interface with the PMU firmware.
* Opcodes and relevant return status codes.
* Pseudocode example to call the interface from MMIO-side.

As previously discussed, because this is not a hardware interface (defined in blob software, although fairly stable over families), this documentation is not within rnndb. However it may be of interest to those researching power, performance or the PMU (PGRAPH) Falcon generally.

### CODE REVIEW

File: docs/nvrm/pmu/ucode-cmds.rst [hunk 1]
```diff
@@ -0,0 +1,295 @@
...
+
+
+Introduction
+============
+
+.. todo:: write me
+
+Sample Implementation
+------------------
+Example of setting up, running and handling potential error or timeout states.
```
Comment (koriakin) [inline docs/nvrm/pmu/ucode-cmds.rst]: Add empty line above, lengthen the '---' to match header length.

File: docs/nvrm/pmu/ucode-cmds.rst [hunk 2]
```diff
@@ -0,0 +1,295 @@
...
+
+Pseudocode:
```
Comment (koriakin) [inline docs/nvrm/pmu/ucode-cmds.rst]: Remove the :: from the next line, replace with 'Pseudocode::', drop one indentation level

**Comment by Echelon9** [comment]
@koriakin Updated.

File: docs/nvrm/pmu/ucode-cmds.rst [hunk 3]
```diff
@@ -0,0 +1,295 @@
+.. _ucode-cmds:
```
Comment (koriakin) [inline docs/nvrm/pmu/ucode-cmds.rst]: These labels are global, you might want to use something more specific

Comment (Echelon9) [reply docs/nvrm/pmu/ucode-cmds.rst]: Revised version pushed to the branch.

**Pull request merged**